trezorlib/tezos.py

```diff
@@ -0,0 +1,112 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2018 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+from binascii import unhexlify
+
+from . import messages
+from .client import field
+from .client import expect
+
+
+@field('address')
+@expect(messages.TezosAddress)
+def get_address(client, address_n, curve, show_display=False):
+    return client.call(
+        messages.TezosGetAddress(
+            address_n=address_n, curve=curve, show_display=show_display))
+
+
+@field('public_key')
+@expect(messages.TezosPublicKey)
+def get_public_key(client, address_n, curve, show_display=False):
+    return client.call(
+        messages.TezosGetPublicKey(
+            address_n=address_n, curve=curve, show_display=show_display))
+
+
+@expect(messages.TezosSignedTx)
+def sign_tx(client, msg: messages.TezosSignTx):
+    return client.call(msg)
+
+
+def create_sign_tx_msg(address_n, curve, transaction) -> messages.TezosSignTx:
```

Comment:
you might be able to replace most of this function with `protobuf.dict_to_proto`